libp2p/multiaddress.nim

```diff
@@ -541,6 +541,17 @@ proc protoArgument*(ma: MultiAddress,
       else:
         ok(res)
 
+proc protoArgument*(ma: MultiAddress): MaResult[seq[byte]] =
```

Comment:
protoArgument here and protoAddress (really bad name) below seems to do the same thing.
Either depecrate one or merge the two procedures into one.

Reply:
Done